kola/tests/fleet/fleet.go

```diff
@@ -124,17 +124,21 @@ func Proxy(c platform.TestCluster) error {
 	}
 	defer proxy.Destroy()
 
-	// settling...
-	time.Sleep(5 * time.Second)
-
 	err = copyFile(proxy, strings.NewReader(fleetunit), "/home/core/hello.service")
 	if err != nil {
 		return fmt.Errorf("copyFile: %s", err)
 	}
 
-	_, err = proxy.SSH("fleetctl start /home/core/hello.service")
-	if err != nil {
-		return fmt.Errorf("fleetctl start: %s", err)
+	// settling...
+	retryFunc := func() error {
```

Comment:
retryFunc is a terrible name... it's already obviously the retry function, it's passed to util.Retry().  But the opportunity to document *what* is being retried in the name has been missed.